src/components/DirectMediaHandler/index.tsx

```
@@ -6,12 +6,22 @@ interface DirectMediaHandlerProps {
-  // Default to video for video extensions or unknown/ambiguous URLs
+  // Treat image formats as images (including GIFs)
+  if (/\.(jpe?g|png|webp|bmp|svg|gif)(\?.*)?$/i.test(u)) return 'image';
+  // Default to video for video extensions or unknown URLs
   return 'video';
 };
 
+const cssUrl = (url: string): string => {
+  return `url("${String(url)
+    // Strip all control characters including line separators
+    // eslint-disable-next-line no-control-regex
+    .replace(/[\x00-\x1F\x7F-\x9F\u2028\u2029]/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)